docs/configuration/index.md

```diff
@@ -1605,7 +1605,8 @@ These Broker configurations can be defined in the `broker/runtime.properties` fi
 |--------|---------------|-----------|-------|
 |`druid.broker.balancer.type`|`random`, `connectionCount`|Determines how the broker balances connections to Historical processes. `random` choose randomly, `connectionCount` picks the process with the fewest number of active connections to|`random`|
 |`druid.broker.select.tier`|`highestPriority`, `lowestPriority`, `custom`|If segments are cross-replicated across tiers in a cluster, you can tell the broker to prefer to select segments in a tier with a certain priority.|`highestPriority`|
-|`druid.broker.select.tier.custom.priorities`|`An array of integer priorities.` E.g., `[-1, 0, 1, 2]`|Select servers in tiers with a custom priority list.|The config only has effect if `druid.broker.select.tier` is set to `custom`. If `druid.broker.select.tier` is set to `custom` but this config is not specified, the effect is the same as `druid.broker.select.tier` set to `highestPriority`. Any of the integers in this config can be ignored if there's no corresponding tiers with such priorities. Tiers with priorities explicitly specified in this config always have higher priority than those not and those not specified fall back to use `highestPriority` strategy among themselves.|
+|`druid.broker.select.tier.custom.priorities`|An array of integer priorities. E.g., `[-1, 0, 1, 2]`|Select servers in tiers with a custom priority list.|The config only has effect if `druid.broker.select.tier` is set to `custom`. If `druid.broker.select.tier` is set to `custom` but this config is not specified, the effect is the same as `druid.broker.select.tier` set to `highestPriority`. Any of the integers in this config can be ignored if there's no corresponding tiers with such priorities. Tiers with priorities explicitly specified in this config always have higher priority than those not and those not specified fall back to use `highestPriority` strategy among themselves.|
+|`druid.broker.select.tier.custom.tierNames`|A non-empty array of tier names, e.g., `["_default_tier","_hot_tier"]`|Specifies the tier names to which the Broker can connect. The Broker cannot connect to a Historical process belonging to a tier not listed in this config.|All tiers are allowed if no value is specified for this config or if an empty array is specified. This config has effect only if `druid.broker.select.tier` is set to `custom`.|
```

Comment:
```suggestion
|`druid.broker.select.tier.custom.allowedTierNames`|A non-empty array of tier names, e.g., `["_default_tier","_hot_tier"]`|Specifies the tier names to which the Broker can connect. The Broker cannot connect to a Historical process belonging to a tier not listed in this config.|All tiers are allowed if no value is specified for this config or if an empty array is specified. This config has effect only if `druid.broker.select.tier` is set to `custom`.|`[]`|
```

Reply:
Fixed the name of the config. The last `[]` is not needed as there are already four columns.